tools/standard-test/integtest_dependencies_opensearch.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# Copyright OpenSearch Contributors.
```

Comment:
Why isn't the complete SPDX header used here?

Ref: https://github.com/opensearch-project/opensearch-plugins/blob/main/HEADERS.md#license-headers

Reply:
It should be, but I am only fixing copyrights here.

We need automation to check headers in this repo, which is https://github.com/opensearch-project/opensearch-build/issues/242